lib/SimpleSAML/IdP.php

```diff
@@ -526,8 +528,11 @@ public function handleLogoutResponse($assocId, $relayState, Error\Exception $err
         assert(is_string($assocId));
         assert(is_string($relayState) || $relayState === null);
 
+        $index = strpos($assocId, ':');
```

Comment:
Perhaps raising an exception if `$assocId` does not meet a certain format would be better.. But what would that format be?